AnalyticsSampleAppExt/FileProviderEnumerator.swift

```diff
@@ -0,0 +1,55 @@
+/*
+ Copyright 2022 Adobe. All rights reserved.
+ This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License. You may obtain a copy
+ of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software distributed under
+ the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ OF ANY KIND, either express or implied. See the License for the specific language
+ governing permissions and limitations under the License.
+ */
+
+import FileProvider
+
+class FileProviderEnumerator: NSObject, NSFileProviderEnumerator {
+
+    var enumeratedItemIdentifier: NSFileProviderItemIdentifier
+
+    init(enumeratedItemIdentifier: NSFileProviderItemIdentifier) {
+        self.enumeratedItemIdentifier = enumeratedItemIdentifier
+        super.init()
+    }
+
+    func invalidate() {
+        // TODO: perform invalidation of server connection if necessary
```

Comment:
There are a lot of TODOs in this sample app. Is this work you'll do in another PR or is this FileProviderExtension a sample taken from someplace else and the actual implementation isn't needed for testing Analytics in an app extension?

Reply:
This is just Apple's template for a File Extension.